### DISCUSSION TITLE
Is possible to mark transactions as"transparent"/"shadow"?

### DISCUSSION THREAD

**Discussion by qRoC** [firefly-iii/firefly-iii#8674] [Firefly III] [answered]
Similar issues: #6075, #5675, #5675, and more other
But merging is so hard, and cannot be automated.

## Use case

A have 2 account `A` and `B`.

I transfer 100$ from `B` to `A`.
Result:
 - Transaction on `B`: -100 to cash, with `external_id=id1`
 - Transaction on `A`: +100 from cash, with `external_id=id2`
 
Based on description from first transaction I can crate rule: `Changed transaction type (transfer)`

Now I have problem: 
  - B account: -100$ (ok)
  - A account: +100$, +100$

Based on description from second transaction I can crate rule: `Delete transaction`
But in this case I will lose the transaction from history

# Proposal

Add new status for transaction - the transaction is shown in history, but is not taken into account when calculating. Additionally, visually highlight it.

Or perhaps other solutions already exist?

**Comment by JC5** [comment, marked as answer]
I don't think I get it. Transaction 2 is a duplicate and contains no useful information in this scenario.
